Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/smart_priority.cpp

```diff
@@ -177,12 +195,20 @@ void smart_priority::msg_add_location(ClassificationItem& item) {
 	int y_top = item.top_left.y;
 	int x_right = item.bottom_right.x;
 	int y_bottom = item.bottom_right.y;
+
+
+	SPDLOG_INFO("x_left = {}", x_left);
+	SPDLOG_INFO("y_top = {}", y_top);
+	SPDLOG_INFO("x_right = {}", x_right);
+	SPDLOG_INFO("y_bottom = {}", y_bottom);
```

Comment:
For conciseness, couldn't these be put into one SPDLOG statement?
like SPDLOG_INFO("x_left: {}, y_top: {} ...")

Reply:
Especially since this is printed for each item right?